### PR TITLE
Homework wk2 nadin

### PR DESCRIPTION
**Pull request by Nadoshka**

### CODE REVIEW

File: week2/homework/src/index.js [hunk 1]
```diff
@@ -1,3 +1,126 @@
...
+        console.error(error);
+      }
+    }
+    else {
+      console.log(JSON.parse(todos));
+    }
+  });
+}
+
+function add() {
```
Comment (gajduk) [inline week2/homework/src/index.js]: separate functions 👍

File: week2/homework/src/index.js [hunk 2]
```diff
@@ -1,3 +1,126 @@
...
+    if (error) {
+      if (error.code === 'ENOENT') {
+        console.log('file not found');
+      }
+      else {
+        console.error(error);
+      }
+    }
+    else {
+      const todoList = JSON.parse(todos);
```
Comment (gajduk) [inline week2/homework/src/index.js]: using JSON 👍

File: week2/homework/src/index.js [hunk 3]
```diff
@@ -1,3 +1,126 @@
...
+      console.log(JSON.parse(todos));
+    }
+  });
+}
+
+function add() {
+  fs.readFile(path, 'utf8', (error, todos) => {
+    if (error) {
+      if (error.code === 'ENOENT') {
+        console.log('file not found');
```
Comment (gajduk) [inline week2/homework/src/index.js]: 👍  error handling

File: week2/homework/src/index.js [hunk 4]
```diff
@@ -1,3 +1,126 @@
...
+    }
+    else {
+      const todoList = JSON.parse(todos);
+      if (!todoItem) {
+        console.log('please write the item you want to add');
+      }
+      else {
+        todoList.push(todoItem);
+        const newList = JSON.stringify(todoList);
+        fs.writeFile(path, newList, 'utf8', () => {
```
Comment (gajduk) [inline week2/homework/src/index.js]: what would happen if you do not have permissions to write to this file?

File: week2/homework/src/index.js [hunk 5]
```diff
@@ -1,3 +1,126 @@
...
+}
+
+function update() {
+  fs.readFile(path, 'utf8', (error, todos) => {
+    if (error) {
+      console.error(error);
+    }
+    else {
+      let todoList = JSON.parse(todos);
+      todoList.splice(todoItem - 1, 1, description);
```
Comment (gajduk) [inline week2/homework/src/index.js]: is there a simpler way to do this?

Comment (Nadoshka) [reply week2/homework/src/index.js]: Thanks for letting me think in another way! :) It's done!

File: week2/homework/src/index.js (hunk 4 above)
Comment (Nadoshka) [reply week2/homework/src/index.js]: It will give me an Error. :) It's done! Thanks!

**Pull request closed without merging**